### PR TITLE
Vet Clinic: Create animals table

### PR DESCRIPTION
**Pull request by mahdinoori2000**
### Added Featured

- [ ] Create vet_clinic database
- [ ] Create animals table
      -  Animals table has below data:
           id: integer
           name: string
           date_of_birth: date
           escape_attempts: integer
           neutered: boolean
           weight_kg: decimal
 

- [ ] Animals table has below queries

     -  SELECT * FROM animals WHERE name LIKE '%mon%';      
     
        query 1

     - SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
     
       query 2

     - SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
     
       query 3

     - SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
     
       query 4

     - SELECT name, escape_attempts FROM animals WHERE weight_kg >10.5;

       query 5

     -  SELECT * FROM animals WHERE neutered = TRUE;
       
        query 06

     - SELECT * FROM animals WHERE name = 'Gabumon';
       
     query 7

    - SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
    
     query 8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +49,59 @@ The Vet Clinic database, created with PostgreSQL, stores essential information a
...
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Add more tables**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
```
Comment (V-Blaze) [inline README.md:107]: - [ ] You have done great so far :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
> Find all animals not named Gabumon.

- [ ] According to the project requirement, you were asked to write a query to select animals not named Gabumon, but it seems you did the opposite. Your current query `SELECT * FROM animals WHERE name = 'Gabumon';` returns animals named Gabumon. I'll suggest you rewrite your query correctly and also update the result screenshot. :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @mahdinoori2000 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized codebase ✔
-  Correct gitflow👋 
-  Readme looks professional ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mahdinoori2000) [reply README.md:107]: ### What is added newly

- SELECT * FROM animals WHERE name != 'Gabumon';

amended query

Comment (Gambit142) [reply README.md:107]: - [ ] Great work writing the query in the terminal but you didn't update this query in the `queries.sql` file. 💪 Kindly update this query. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @mahdinoori2000 👋, 

Great work on making the changes requested by a previous reviewer 👏 

### Highlights
- Everything that creates or changes the structure of the table was saved in the _**schema.sql**_ file ✔️
- Everything that populates tables with data was saved in the _**data.sql**_ file ✔️ 
- Everything that is a query that helped to analyze data was saved in the _**queries.sql**_ file ✔️

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @mahdinoori2000, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**